tests/annotator/test_infection_annotator.py

```diff
@@ -20,15 +20,8 @@ def setUp(self):
     def assertHasCounts(self, sent, counts):
         doc = AnnoDoc(sent)
         doc.add_tier(self.annotator)
-        actuals = []
-        expecteds = []
-        for actual, expected in zip(doc.tiers['infections'].spans, counts):
-            if expected.get('count'):
-                actuals += [actual.metadata.get('count')]
-                expecteds += [expected.get('count')]
-            else:
-                actuals += [None]
-                expecteds += [None]
```

Comment:
I'm not sure why I did this this way (I should have commented why), but there was a reason I didn't just use a list comprehension to get all the count like that.

Reply:
@toph-allen The old version allows the presence of a count and its other metadata to be specified without having to exactly define its value. We didn't really need that feature so I decided to go with something simpler. However, the reason I replaced this code was not for that, but rather because zip will only iterate up to the end of the shorter list so there will be no error if the actuals have extras item or are missing the final items from the expecteds.